common/models/create_task_request.go

```diff
@@ -35,6 +36,8 @@ func (c CreateTaskRequest) Definition() swagger.Definition {
 			"task_name":      swagger.NewStringProperty(),
 			"environment_id": swagger.NewStringProperty(),
 			"deploy_id":      swagger.NewStringProperty(),
+			// TODO: Change to Map defintion
```

Comment:
Is this still an intended TODO?

Reply:
Yes it is. @zpatrick suggested it until we add maps to swagger